content/response-to-reviewers.md

```diff
@@ -173,7 +173,7 @@ Same for inline comments and diff colorization.
 
 We added `WYSIWYG mode`, `Inline comments`, and `Viewing changes` as new features to the table that compares collaborative writing platforms.
 In addition, we updated the table to use Overleaf v2 instead of Overleaf v1, which was discontinued.
-We discussed these changes in [GH147](https://github.com/greenelab/meta-review/issues/147) and edited the table in [GH188](https://github.com/greenelab/meta-review/pull/188).
+We discussed these changes in [GH147](https://github.com/greenelab/meta-review/issues/147) and edited the table in [GH188](https://github.com/greenelab/meta-review/pull/188) and [GH196](https://github.com/greenelab/meta-review/pull/196).
```

Comment:
Can you also please list this PR in the preceding reviewer comment about `they lack sufficient features for managing a collaborative editing`?  Your new paragraph better explains the difference between Manubot and other tools and gives existing tools their due credit.